kitsune/wiki/jinja2/wiki/document.html

```diff
@@ -73,9 +74,8 @@
       </div>
     </header>
 
-    {% if show_cta_banner %}
-      {{ sumo_cta_banner(text=_('Unlock more from the Firefox suite: Sync, Monitor, Lockwise and <a href="https://www.mozilla.org/firefox/accounts/">more.</a>'),
-                         modifier_class="hide-on-mobile") }}
+    {% if banner %}
```

Comment:
similar to the comment I made in the view, let's rename this to cta_banner or something more descriptive